src/payment/strategies/paypal-commerce/paypal-commerce-script-loader.ts

```diff
@@ -17,12 +17,14 @@ export default class PaypalCommerceScriptLoader {
     async loadPaypalCommerce(params: PaypalCommerceScriptParams, isProgressiveOnboardingAvailable?: boolean): Promise<PaypalCommerceSDK> {
         this._validateParams(params, isProgressiveOnboardingAvailable);
 
-        const scriptSrc = 'https://unpkg.com/@paypal/paypal-js@1.0.2/dist/paypal.browser.min.js';
+        if (!this._window.paypalLoadScript) {
```

Comment:
@helga1507 so in the case when banners loaded the SDK we do not load SDK here?

Reply:
@helga1507 can you take a look?

Reply:
Yes. We don't need do load this script again

Reply:
I see there is no currency in the banner script, I believe if we don't load this script again there will be an issue because the currency is the mandatory param for button rendering (the possible method that will be shown on the frontend depends on the currency of on the checkout)

Reply:
We load `unpkg` only for function `paypalLoadScript`. All parameter for script we add and call here `await this._window.paypalLoadScript(params);` ([line 30](https://github.com/bigcommerce/checkout-sdk-js/blob/5c75f24d892c1234852780151cd8a7c492cab0c0/src/payment/strategies/paypal-commerce/paypal-commerce-script-loader.ts#L30))

Reply:
@helga1507 can you break it down for me?https://github.com/bigcommerce/content/pull/615/files#diff-544247c5f4bda100deda99d1efa9c7626bf30376509651c75a1c46c47e507f23R24 we do not pass currency here, hence,  if the banner loaded before the SPBs, SPBs will consume the initialized SDK without currency that has been loaded before in the banner, is it correct?

Reply:
Per our discussion, such an approach makes sense for the new version of the `unpkg `